pwem/protocols/protocol_align_movies.py

```diff
@@ -105,6 +105,16 @@ def _defineAlignmentParams(self, form):
                       label="Save aligned micrograph",
                       expertLevel=pwcts.LEVEL_ADVANCED)
 
+        form.addHidden('tomographySplitFrames', params.BooleanParam,
```

Comment:
I think tomography stuff should not be at this level...if base align movies protocol is needed, maybe a new class or a sub-class of this one.